ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Reserve more system memory per node than a typical multi-node cluster
+# to facilitate E2E tests on a single node
+# set shutdown grace period toenable testing graceful shutdown
+cat > "${SHARED_DIR}/manifest_single-node-reserve-sys-mem-kubeletconfig.yml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: single-node-reserve-sys-mem
+spec:
+  containers:
+    machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/master: ""
+  kubeletConfig:
+    shutdownGracePeriod: 600s
+    shutdownGracePeriodCriticalPods: 300s
+    systemReserved:
```

Comment:
Still defines systemreserved

Reply:
we still want to define systemReserved, no? There is one kubelet config.

Reply:
Already brought it up in https://github.com/openshift/release/pull/20848#discussion_r685456683

If definines multiple kubeletconfigs is not possible, then the second bullet there is irrelevant. But on another file in this PR @wgahnagl did assume that multiple kubeletconfigs are possible (possibly because they ignored my "Does MCO support merging" remark from the comment linked above).

Now for some reason this PR defines this file twice, once with system reserved, once without, and the name has aws in it in one of them but it has nothing to do with aws...

I've addressed all those problems in my other different comments.

Anyway if merging is not supported then I suggest that the one step should modify the kubeletconfig created by the other, rather than duplicating the entire kubelet config